java/ql/test/utils/model-generator/CaptureSummaryModels.expected

```diff
@@ -14,6 +14,8 @@
 | p;InnerHolder;false;getValue;();;Argument[-1];ReturnValue;taint |
 | p;InnerHolder;false;setContext;(String);;Argument[0];Argument[-1];taint |
 | p;Joiner;false;Joiner;(CharSequence);;Argument[0];Argument[-1];taint |
+| p;Joiner;false;Joiner;(CharSequence);;Argument[1];Argument[-1];taint |
```

Comment:
This can't be right; `Argument[1]` does not exist for a constructor with just one argument.

Reply:
Hm, yes. Good point. Since you've fixed this in https://github.com/github/codeql/pull/8474 I won't dive into why that happened (but I'm still kinda curious 🤔).